gcore/gdaldriver.cpp

```diff
@@ -1063,8 +1063,7 @@ CPLErr GDALDriver::QuietDeleteForCreateCopy(const char *pszFilename,
                 {
                     CPLString osFilename(pszFileInList);
                     osFilename.replaceAll('\\', '/');
-                    if (oSetExistingDestFiles.find(osFilename) !=
-                        oSetExistingDestFiles.end())
+                    if (cpl::contains(oSetExistingDestFiles, osFilename))
                     {
                         oSetExistingDestFilesFoundInSource.insert(osFilename);
                     }
```

Comment:
```suggestion
                    oSetExistingDestFilesFoundInSource.insert(osFilename);
```

Reply:
no, I actually confused myself initially doing that "optimization" :-), but oSetExistingDestFiles and oSetExistingDestFilesFoundInSource are distinct sets ...